src/test/java/org/eolang/speco/SpecoTest.java

```diff
@@ -131,6 +136,29 @@ public void compilesFromEo(final String pack, @TempDir final Path temp)
         );
     }
 
+    /**
+     * Unit tests for transformations.
+     * @param pack Pack with test data
+     * @throws IOException Iff IO error
+     */
+    @Tag("fast")
+    @ParameterizedTest
+    @ClasspathSource(value = "org/eolang/speco/transformations", glob = "**.yaml")
+    public void applyTransToXmir(final String pack) throws IOException {
+        final Map<String, Object> script = new Yaml().load(pack);
+        MatcherAssert.assertThat(
+            "Unexpected transformation result",
+            new Xsline(
+                new TrDefault<Shift>()
+                    .with(new StClasspath("/org/eolang/parser/wrap-method-calls.xsl"))
+                    .with(new StClasspath(script.get("xsl").toString()))
+            ).pass(new XMLDocument(script.get("before").toString())),
+            Matchers.equalTo(
+                new XMLDocument(script.get("after").toString())
```

Comment:
@advasileva comparing XML documents one-to-one is not a safe testing method. The output format may slightly be changed in the future (for example by additional indentation or a different style of formatting) and all tests will be invalidated. It's better to use XPath to check the output XML. I suggest you take a look at this: https://github.com/objectionary/xax

Reply:
@yegor256 sorry, I don't understand how to write an `assert` to compare two nodes for equality. I found `count(/program/speco | /program/aoi) = 1`, but it produces an exception `IllegalArgument Invalid XPath query`

Full test:
```yaml
sheets:
  - /org/eolang/speco/1-1-coping.xsl
document:
  <?xml version="1.0" encoding="UTF-8"?>
  <program>
     <listing/>
     <errors/>
     <sheets/>
     <objects/>
     <aoi>
        <obj fqn="foo.x">
           <inferred>
              <obj fqn="first"/>
              <obj fqn="second"/>
           </inferred>
        </obj>
     </aoi>
  </program>
asserts:
  - count(/program/speco | /program/aoi) = 1
 ```